documentation/automation.md

```diff
@@ -33,7 +33,7 @@
 | Regression                   | `regression_org`     | `release`        | ✔       |           |
 | Latest Trial Template        | N/A                  | `trial`          | ✔       |           |
 | Update Trialforce Source Org | `trial_org`          | `release`        | ✔       |           |
-| Upgraded Org                 | `upgraded_org`       | `release`            |         |           |
+| Upgraded Org                 | `upgraded_org`       | `release`        |         |           |
```

Comment:
Since you're updating this, can you add a `✔` to the `Managed` column for this flow?